server/cli/launch.py

```diff
@@ -199,7 +212,7 @@ def launch(
         sys.stdout = f
 
     try:
-        server.app.run(host=host, debug=debug, port=port, threaded=True)
+        server.app.run(host=host, debug=developer, port=port, threaded=True, use_reloader=developer, use_debugger=False)
```

Comment:
```suggestion
        server.app.run(host=host, debug=debug, port=port, threaded=True, use_reloader=developer, use_debugger=False)
```
I think this was just a typo or oversight